### DISCUSSION TITLE
Abogabot Finished

### DISCUSSION THREAD

**Discussion by DiegoCorchuelo** [Launch-X-Latam/MisionFrontEnd#145] [Show and tell]
Comenten que opinan y en que puedo mejorar, aquí dejo mi repositorio. Gracias 
https://github.com/DiegoCorchuelo/Abogabot

**Comment by Alejandro-Ari** [comment]
Se ve muy padre tu trabajo!!!

En donde hiciste tu UI/UX?

Saludos.

**Reply by DiegoCorchuelo** [reply]
Muchas gracias Alejandro, El UI/UX lo hice en Balsamiq.
Saludos.

**Reply by KevAguirre** [reply]
El wireframe en one note? el bosquejo.

**Reply by KevAguirre** [reply]
![image](https://user-images.githubusercontent.com/62616097/194166839-fec257f2-6486-445f-8838-3c6278ed5ebc.png)

**Reply by DiegoCorchuelo** [reply]
Eso lo hice en https://excalidraw.com/
Espere te sirva, es muy fácil de usar .

**Comment by AndrjsM** [comment]
Buen diseño bro, bastante simple.

Importante considerar el responsive de la página, diseño mobile y desktop.

Saludos.

**Reply by DiegoCorchuelo** [reply]
Muchas gracias, Saludos.

**Comment by DaveOval** [comment]
Muchas gracias por compartir tu proyecto se ve muy padre , saludos.

**Reply by DiegoCorchuelo** [reply]
Muchas gracias, Saludos.

**Comment by T0ny-dev** [comment]
buena bro , gracias por compartir y las herramientas
